native/react-native-bpk-component-text/index.js

```diff
@@ -0,0 +1,3 @@
+import BpkText from './src/BpkText';
```

Comment:
Shouldn't all new files have the license header?

Reply:
Good catch, fixed